Bazel commands only work in the magma directory

Since #12987 was merged, the bazel commands will only work in the magma folder, because the cache setup was reworked to use a relative path: `bazel/bazelrcs/cache.bazelrc`

This also causes some make targets to fail (such as `build_c`) because they invoke bazel form `lte/gateway`.
```
vagrant@magma-dev-focal:~/magma/lte/gateway$ make build_c
BAZEL_FLAGS 
MAIN_FEATURE agw_of
OAI_FLAGS -DS6A_OVER_GRPC=True -DEMBEDDED_SGW=True
bazel build  //lte/gateway/c/session_manager:sessiond //lte/gateway/c/sctpd/src:sctpd //lte/gateway/c/connection_tracker/src:connectiond //lte/gateway/c/li_agent/src:liagentd
[INFO 09:42:48.881 src/main/cpp/option_processor.cc:407] Looking for the following rc files: /etc/bazel.bazelrc,/home/vagrant/magma/.bazelrc,/home/vagrant/.bazelrc
[INFO 09:42:48.881 src/main/cpp/rc_file.cc:56] Parsing the RcFile /home/vagrant/magma/.bazelrc
[INFO 09:42:48.881 src/main/cpp/rc_file.cc:56] Parsing the RcFile bazel/bazelrcs/cache.bazelrc
[FATAL 09:42:48.881 src/main/cpp/blaze.cc:1295] Unexpected error reading .blazerc file 'bazel/bazelrcs/cache.bazelrc'
make: *** [Makefile:154: build_c] Error 37
```

--- a/.bazelrc
+++ b/.bazelrc
@@ -48,7 +48,7 @@ run:remote_caching_ro --remote_download_toplevel
 
 # Import the bazel caching config, with default disk cache,
 # which in CI gets overwritten by the remote caching config. 
-import bazel/bazelrcs/cache.bazelrc
+import %workspace%/bazel/bazelrcs/cache.bazelrc
 
 # TEST CONFIGS
 # Bazel test runtime default: PATH=/bin:/usr/bin:/usr/local/bin

